### PR TITLE
DOC: add release notes for v3.0.0

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

- add release notes using the script
- add a pre-merge-checklist to the issue template

I thought the API changes were enough to warrant a v3.0.0
As usual, reviewers are everyone who contributed to the tag.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,3 +18,9 @@
 <!--
 ## Screenshots (if appropriate):
 -->
+
+## Pre-merge checklist
+- [ ] Test suite passes locally
+- [ ] Test suite passes on travis
+- [ ] Code works interactively
+- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
```
Comment (klauer) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: A ➕

**Review by klauer — APPROVED** [review]
Bigger release than I recalled. Nice - v3.0.0.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (cristinasewell) [reply .github/PULL_REQUEST_TEMPLATE.md:26]: 👏

**Review by cristinasewell — COMMENTED** [review]
A bit late for the review, but looks great and It is good to know what goes into the release notes 👍 
One thing that I think was wrong on my part, is to say that there were `API changes` related to the `alio_to_theta` and `theta_to_alio` calculations, because theoretically we still have the same inputs and outputs - sorry about that...

**Comment by ZLLentz** [comment]
I think the output is different enough for it to count as an API change. As in, if you ran analysis with the calculation code somehow, then you'd get a different result. So I thought the categorization was :+1:

This release note amalgamator is a bit of an experiment right now, I think it's working nicely.

Also, consider release note review requests as a "hey, your code is being released!" more than anything else. (especially here, where we're essentially copy/pasting already-reviewed documentation)

**Pull request merged**